lexers/LexPython.cxx

```diff
@@ -321,6 +322,9 @@ struct OptionSetPython : public OptionSet<OptionsPython> {
 		DefineProperty("lexer.python.strings.f", &OptionsPython::stringsF,
 			       "Set to 0 to not recognise Python 3.6 f-string literals f\"var={var}\".");
 
+		DefineProperty("lexer.python.strings.f.pep.701", &OptionsPython::stringsF,
```

Comment:
`stringsF` is typo for `pep701StringsF`?

Reply:
Fixed in commit.